composer.json

```diff
@@ -77,7 +77,7 @@
         "composer/composer": "2.*",
         "visiosoft/composer-merge-plugin": "2.*",
         "guzzlehttp/guzzle": "^7.3",
-        "visiosoft/connect-module": "^1.0",
+        "visiosoft/connect-module": "1.10.9",
```

Comment:
why we fixed it @erhanask

Reply:
dispatch function usages were changed to dispatchSync and it was not compatible for some reason as far as I remember. We don't need this anymore.